checks/test.go

```diff
@@ -131,7 +132,23 @@ func (t *T) markConfigured() {
 	// require caves should be the marker, and I believe that access to t.Mock()
 	// should be guarded by that feature.
 
-	t.cluster = t.parent.defaultCluster
+	var cluster mock.Cluster
```

Comment:
This doesn't seem right

Reply:
There will likely be a number of reasons that we need to use a custom cluster, instead of looking for a particular feature, we should have a "RequiresMock" function call during configuration of a check which triggers this (so the test is in control, rather than a feature flag).